color_top3:
 upper_red: [0, 0, 0]

 lower_red: [255, 255, 255]

 upper_green: [0, 0, 0]

 lower_green: [255, 255, 255]

 upper_yellow: [0, 0, 0]

 lower_yellow: [255, 255, 255]

 upper_orange: [0, 0, 0]

 lower_orange: [255, 255, 255]